ansible/playbooks/install_python.yml

```diff
@@ -9,6 +9,14 @@
 
   tasks:
 
+   - debug:
+       msg: "Running playbook as {{ lookup('env', 'USER') }} ..."
+
    - name: Install Python
      sudo: yes
-     shell: cd {{ app_dir }}; sh bin/install_python3_5.sh >> dev_build.log
+     sudo_user: "{{ lookup('env', 'USER') }}"
```

Comment:
In previous case commands were executed as `root`, but probably most users of playbok will want to install all stuff as their own user as we don't create any project specific users. 
This will get current user and use it as a sudo user.